examples/with-typescript/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "scripts": {
+    "dev": "concurrently 'tsc --watch' 'next'"
+  },
+  "dependencies": {
+    "next": "latest"
+  },
+  "devDependencies": {
+    "@types/react": "^15.0.1",
+    "babel-preset-react": "^6.22.0",
```

Comment:
I thought I was getting an error without it... but I have tried again and it's not needed.
Removed.